### ISSUE TITLE
Some metrics show up twice in metrics_status

### ISSUE THREAD

**Issue by parthsharma2** [chaoss/augur#231]
**Describe the bug:**
Metrics implemented in `ghtorrent` show up twice in "Metrics Status." The same thing seems to be happening with certain metrics in `githubapi`. In `githuapi` certain metrics like "bus factor", "tags", etc. show up twice while metrics like "Lines of code changed" show up once.

**To Reproduce:**
1. Start the Flask server.
2. Open the web application.
3. Select "Metric Status" form the navbar.
4. Select "ghtorrent" as the source.
5. See the duplicate metrics statuses.

**Expected behavior:**
The metric statuses should show up only once per metric.

**Screenshots**
![ghtorrent screenshot 1](https://i.imgur.com/a9GaCq2.png)
![ghtorrent screenshot 2](https://i.imgur.com/sYPVrBz.png?1)
![ghtorrent GIF](https://i.imgur.com/a8sYsHj.gif)
![githubapi screenshot](https://i.imgur.com/h9luhFX.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04
 - Browser: Chrome Version 71.0.3578.80

**Comment by ccarterlandis** [comment]
I accidentally did this while trying to fix another issue in metrics_status. I'm rolling back metrics status to an earlier version while I try to fix the actual issue.

**Comment by ccarterlandis** [comment]
Fixed in dev as of [this](https://github.com/chaoss/augur/commit/d65c1fa6538d03aa7e8030f96253934ecd57b5aa) commit.
